packages/components/src/components/Textarea/index.tsx

```diff
@@ -0,0 +1,120 @@
+import React, { useState, useCallback } from 'react';
+import styled from 'styled-components';
+import css from '@styled-system/css';
+import { VisuallyHidden } from 'reakit/VisuallyHidden';
+import { uniqueId } from 'lodash-es';
+import { Text } from '../Text';
+import { Stack } from '../Stack';
+
+const placeholderStyles = {
+  color: 'input.placeholderForeground',
+  fontSize: 13,
+};
+
+export const TextareaComponent = styled.textarea(
+  css({
+    height: 64,
+    width: '100%',
+    padding: 2,
+    fontSize: 3,
+    resize: 'none',
+    backgroundColor: 'input.background',
+    borderBottom: '1px solid',
+    borderColor: 'input.border',
+    color: 'input.foreground',
+    borderRadius: 'small',
+    '::-webkit-input-placeholder': placeholderStyles,
+    '::-ms-input-placeholder': placeholderStyles,
+    '::placeholder': placeholderStyles,
+  })
+);
+
+const Label = styled(Text)(
+  css({
+    marginBottom: 2,
+    display: 'block',
+  })
+);
+
+const Count = styled.div<{ limit: boolean }>(({ limit }) =>
+  css({
+    fontSize: 11,
+    paddingTop: 1,
+    color: limit ? 'errorForeground' : 'input.placeholderForeground',
+    alignSelf: 'flex-end',
```

Comment:
interesting way to use this, hadn't thought of that!

Reply:
Flex is amazing! ahah